src/sdks/manage/sdk.config.json

```diff
@@ -29,7 +29,8 @@
                 "xrn:firebolt:capability:localization:country-code",
```

Comment:
We should probably add this to `use` as well, so that Manage Apps can check the value before setting it